### ISSUE TITLE
adding points in the devices

### ISSUE THREAD

**Issue by gioassio89** [Bepacom-Raalte/Bepacom-BACnet-IP-Integration#25]
Hello, I noticed that every time I export a new point to BACnet from a device, I have to delete the entire device from the integration and then reload it to make the device reappear with the new point. Is there another way to update the list of points without having to delete the device each time? For example: a "reconfigure" button present in many integrations. Thank you.

**Comment by gioassio89** [comment]
Yes, sorry, I made a mistake: I should have written this request in the add-on. If you want, I'll close it here and open it in the right place. Anyway, yes, I'm finalizing the programming of two devices. The ultimate goal would be to display some parameters of both devices on HA to have an interface easily accessible from a smartphone. With one of the two devices, I can export the variables I want in BACnet, so I can already make a selection from there. But with the other one, I need to create a subset of BACnet variables in Home Assistant. I could import the device into the add-on with all the variables disabled and then enable only the ones I need. But if I were to add a new point to the device that I need to import, I would be forced to delete the entire device and redo the selection from scratch, or at least that's what I saw from some tests I did.

**Comment by GravySeal** [comment]
I think the issue for the add-on is that whenever you updated your BACnet device's configuration, the add-on still assumes the old objects are there. It doesn't check for new objects, so it'll never update to the new configuration.
If you want, it might be useful to start a new issue at the add-on GitHub. That way I'll keep it in mind when working on the add-on.

**Comment by gioassio89** [comment]
ok i'll open a new issue there, thank you!
